src/newrelic/limits/limits.go

```diff
@@ -34,7 +34,7 @@ const (
 	DefaultReportPeriod          = 60 * time.Second
 	MaxMetrics                   = 2 * 1000
 	MaxTxnEvents                 = 10 * 1000
-	MaxCustomEvents              = 10 * 1000
+	MaxCustomMaxEvents           = 100 * 1000
```

Comment:
Not something to be addressed before merge but somehow `MaxCustomMaxEvents` with two `Max` does not clearly convey the purpose of this constant. I guess it has been following `MaxSpanMaxEvents` and `MaxLogMaxEvents` but there also is `MaxTxnEvents` which IMHO better matches the intention.